src/Pos/Context/Functions.hs

```diff
@@ -4,6 +4,7 @@ module Pos.Context.Functions
        (
          -- * Genesis
          GenesisUtxo(..)
+       , mkGenesisTxpContext
```

Comment:
otherwise, I would be forced to add `cardano-sl-txp` to block-gen or something like that.
this functions more or less related to `Context`